docs/pages/stacks.md

```diff
@@ -59,11 +59,19 @@ npx create-remix@latest --template your-private/repo --token yourtoken
 
 The [token just needs `repo` access][repo access token].
 
+### Community Templates
+
+Community-provided templates can help you get started with other platforms or ideas. These are not owned or maintained by Remix.
+
+#### Platforms
+
+- [Remix + Fastly Compute@Edge][fastly-remix-template]: Deployed to [Fastly's Compute@Edge][fastly-compute-at-edge].
+
```

Comment:
Not sure if we want this in our docs, as that could cause a list of every possible stack out there 🤔

CC/ @mcansh @brophdawg11

Reply:
I'm not sure either.

Let's stick to adapters for now please 🙏🏼